api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -435,4 +436,25 @@ public void updateDbRdrExport(RdrEntity entity, List<Long> idList) {
             .collect(Collectors.toList());
     rdrExportDao.save(exportList);
   }
+
+  /**
+   * Set excludeFromPublicDirectory to true if the workspace creator is an operational user i.e has
+   * Institution as All of Us Program operational Use
+   *
+   * @param creatorUser
+   * @param rdrWorkspace
+   */
+  void setExcludeFromPublicDirectory(DbUser creatorUser, RdrWorkspace rdrWorkspace) {
+    rdrWorkspace.setExcludeFromPublicDirectory(false);
+    verifiedInstitutionalAffiliationDao
+        .findFirstByUser(creatorUser)
+        .ifPresent(
+            verifiedInstitutionalAffiliation -> {
+              rdrWorkspace.setExcludeFromPublicDirectory(
+                  verifiedInstitutionalAffiliation
+                      .getInstitution()
+                      .getShortName()
+                      .equals("AouOps"));
```

Comment:
Please make this a file-level constant. Or, better yet, most of this method might be better placed within the institution service, e.g. `institutionService.isOperationalUser(dbUser user)`. That would centralize the logic within the institution package, and I don't think it would require adding any new deps to the institution service.

Slightly tangential thought: I'm concerned that we're at risk of putting load-bearing system weight on the `shortName` identifier, which can be changed by operational end-users via the Institution UI at any time. This is a pretty serious red flag, and we should think about ways to make sure far-flung business logic can't be impacted by someone deciding in an admin UI that "AouOps" should be capitalized differently, or have a space added, etc. etc.

I filed https://precisionmedicineinitiative.atlassian.net/browse/RW-4844 to track – I'd suggest following up with Joel & Karthik to align on a solution for this.